src/cli/upload.js

```diff
@@ -44,18 +44,13 @@ module.exports = function(config, workflowDir) {
      * @returns {import('http').ClientRequest}
      */
     function updateWorkflow(isCreate) {
-      const content = JSON.stringify({
-        name: isCreate ? workflowName : undefined,
-        base64Content: fs.readFileSync(zip.path).toString('base64')
-      });
+      const form = new FormData();
+      form.append('file', fs.createReadStream(zip.path), {filename:workflowName + '.zip'});
 
-      let message = HttpMessage(resolve(config.host, '/api/admin/workflows/' + (isCreate ? '' : encodeURIComponent(workflowName))));
+      let message = HttpMessage(resolve(config.host, '/api/admin/workflows/import' + (isCreate ? '' : encodeURIComponent(workflowName))));
```

Comment:
I believe, we can go even further and use apps endpoint?

```
POST api/admin/apps/import
```

Worth mentioning that there should be no changed done to path for subsequential Apps uploading: the URL is always the same

Reply:
```POST /api/admin/workflows/import``` is what's used in the current YouTrack UI when importing an app.
If ` POST api/admin/apps/import` also will allow for the import of standard workflows, then that could be used too. However if it ONLY supports uploading apps, then it should either be split into its own command/option, or not used.

Reply:
Turns out this apps API is not yet actually released, so we need to stick to Workflow API anyway : )

@TPG-Teklada Anyway we shouldn't change the URL for updating existing workflows I believe, it should be always `/api/admin/workflows/import` without adding workflow name on the end

Reply:
@andrey-skl , ahh good point! I'll remove the workflow name and update the PR